es6/index.js

```diff
@@ -70,22 +70,38 @@ export default class Confluency {
   }
 
 
+  createQueryString(parameters) {
+    Object.keys(parameters).forEach((key) => {
+      if (Array.isArray(parameters[key])) {
+        parameters[key] = parameters[key].join(',');
+      }
+
+      if (!parameters[key] && typeof parameters[key] !== "number") {
+        delete parameters[key];
+      }
```

Comment:
It is to delete items from the query string.  Let's take this example:

    createQueryString({
        expand: undefined,
        limit: 500
    });

That should make a query string of "?limit=500" If you leave the `expand` key in the object, it will return "?expand=&limit=500" instead. So, if the value is falsy but it is not a number, then we want to include it in the query string.